charts/opentelemetry-demo/values.yaml

```diff
@@ -403,6 +403,7 @@ components:
         value: "4318"
     resources:
       limits:
+        # For AMD64 the memory consumption is higher adjust accordingly in case if you run into OOMKilled
```

Comment:
We should just set this to 65Mi like the demo repo now has it set and remove the comment.